lib/inspec/input.rb

```diff
@@ -334,7 +335,7 @@ def to_hash
     #--------------------------------------------------------------------------#
 
     def to_s
-      "Input #{name} with #{current_value}"
+      "Input #{name} with " + (sensitive ? "***" : "#{current_value}")
```

Comment:
The grammar here was already a little terse, it should have said `with value` instead of just `with`.
Now we handle the sensitive case it would be helpful to make this clearer.

Suggest that for non-sensitive it should look like `Input php_username with value php_admin` and for the sensitive case `Input php_password with value *** (sensitive)`

I was also wondering if it would be helpful to handle the case where the input is sensitive by has no value. In that case it would be misleading to show *** as that suggests there is a value but it's been redacted.
However I'm not sure on the use case for that because if I try to include a value-less input in a control, the control gets skipped.

Reply:
OK, I've added "with value" and "(sensitive)".